### ISSUE TITLE
Circuit board variations

### ISSUE THREAD

**Issue by ffraenz** [ffraenz/redoid#2]
![control-unit](https://cloud.githubusercontent.com/assets/1041468/14026264/59c19de4-f1f2-11e5-82ae-3eb507588dcc.jpg)

**Comment by kennylmay** [comment]
![img_20161023_172718](https://cloud.githubusercontent.com/assets/7873950/19629573/18552b54-9946-11e6-872d-7ab838f25fbf.jpg)

You think there is any chance that the same pin-out numbers match on the new version of the board?

**Comment by ffraenz** [comment]
Yes there is!

This configuration worked for me:
- Disconnect the micro controller (1) from the circuit board by destroying the connections on one side or by removing it completely.
- Connect `J1` (2) to `GND`
- Connect `R4` (R) to `GPIO4`
- Connect `R5` (G) to `GPIO17`
- Connect `R6` (B) to `GPIO18`

![img_2796](https://cloud.githubusercontent.com/assets/1041468/19629673/4791eaae-997a-11e6-8842-0806e5bc1ad9.JPG)

**Comment by kennylmay** [comment]
That's great! Thanks so much for the quick reply!

**Comment by ffraenz** [comment]
@kennylmay May I use your photo of the new board to add this configuration to the README?

**Comment by kennylmay** [comment]
Sure thing. Worked great by the way. Thanks for your help!

**Comment by ffraenz** [comment]
![board2edited](https://cloud.githubusercontent.com/assets/1041468/20233401/7ad2ad82-a86f-11e6-8494-324e583f27c9.jpg)

**Comment by thormartin91** [comment]
I have a different version of the board. Any idea which pins I should use?

![ikea_dioder_kzq1m](https://cloud.githubusercontent.com/assets/4323938/23694893/b7f6c626-03dc-11e7-8f66-4ba10b661b1d.jpg)

**Comment by ffraenz** [comment]
There is no color wheel on this control unit. Are you sure this is a multi-color version of Ikea Dioder?

**Comment by thormartin91** [comment]
Yes, it is @ffraenz. A push on the button changes color. Holding down the button activates different modes that loop through the colors.

**Comment by Ancient** [comment]
And another Board Variation.  Any idea which pins to use?
![dsc_0192](https://cloud.githubusercontent.com/assets/232153/24201452/5b1fa514-0f10-11e7-85d7-f99809a96af4.JPG)

**Comment by ffraenz** [comment]
@thormartin91 @Ancient
I guess someone needs to try out different configurations with the possible risk of losing the device. You may recognise similarities when comparing the two known board versions with the new ones. I'm rather working software-side, I can't give you safe advice on this.

**Comment by Ancient** [comment]
@ffraenz 
Hi,
i figured it out for my board Layout, maybe you could add it with your Markers, if someone else needs it.

![dsc_0225](https://cloud.githubusercontent.com/assets/232153/24789558/e270826c-1b73-11e7-9ee2-776086272192.JPG)
